pkgs/applications/networking/browsers/elinks/default.nix

```diff
@@ -1,7 +1,10 @@
-{ stdenv, fetchurl, perl, ncurses, xlibsWrapper, bzip2, zlib, openssl
-, spidermonkey_1_8_5, gpm
-, enableGuile ? false, guile ? null   # Incompatible licenses, LGPLv3 - GPLv2
-, enablePython ? false, python ? null
+{ stdenv, fetchurl, ncurses, xlibsWrapper, bzip2, zlib, openssl
+, gpm
+, # Incompatible licenses, LGPLv3 - GPLv2
+  enableGuile        ? false,                                         guile ? null
+, enablePython       ? false,                                         python ? null
+, enablePerl         ? (stdenv.hostPlatform == stdenv.buildPlatform), perl ? null
+, enableSpidermonkey ? (stdenv.hostPlatform == stdenv.buildPlatform), spidermonkey_1_8_5 ? null
```

Comment:
Ehh. Why was there a need to add these conditionals. If the cross-compilation for this package has been broken for years without anyone noticing, then better to drop the crossAttrs instead of adding conditionals like this (Unless you actually personally want/need this package of course...).

Reply:
I've done this a few times earlier. On one hand while dependencies may also be broken I can't test the package so I just don't want to rip out functionality. On the other, missing deps was likely to be the reason they were disabled in the first place. When written like this, it's very easy to later make things unconditional if they prove unnecessary.